Snowplow iOSTests/TestTracker.m

```diff
@@ -89,6 +91,8 @@ - (void)testTrackerBuilderAndOptions {
     XCTAssertEqual([tracker trackerNamespace], @"newNamespace");
     [tracker setBase64Encoded:YES];
     XCTAssertEqual([tracker base64Encoded], YES);
+    [tracker setDevicePlatform:@"newPlatform"];
```

Comment:
This new test now generates two warning:

`TestTracker.m:94:32: Incompatible pointer to integer conversion sending 'NSString *' to parameter of type 'SPDevicePlatform' (aka 'enum SPDevicePlatform')`

and 

`TestTracker.m:95:5: Comparison between pointer and integer ('typeof ([tracker devicePlatform])' (aka 'enum SPDevicePlatform') and 'typeof (@"newPlatform")' (aka 'NSString *'))`